pinot-segment-local/src/main/java/org/apache/pinot/segment/local/indexsegment/mutable/MutableSegmentImpl.java

```diff
@@ -319,12 +319,16 @@ public long getLatestIngestionTimestamp() {
 
       // Text index
       MutableTextIndex textIndex;
+      List<String> stopWordsInclude = null;
+      List<String> stopWordsExclude = null;
```

Comment:
Minor: it seems like this is the only reason why a null check is needed in `getStandardAnalyzerWithCustomizedStopWords` for `stopWordsInclude` and `stopWordsExclude`.  Otherwise, all the other places that create stop world list are returning non null `List<String>` and marked as `NonNull`.

Reply:
Oh the null checking is for easier testing of `LuceneTextIndexCreator` as well